From 149b0c204c6550a628e0ae76cc027ad760c55c54 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 9 Oct 2019 16:36:46 +0200
Subject: [PATCH 2/2] appdata: add release tag

The description is not filled in. We can't merge this with the NEWS file
because this pertains only to the editor.

There's no machinery to automatically check this yet. Just the bare
minimum to keep the new version of appdata-util happy.
---
 nm-connection-editor.appdata.xml.in | 52 +++++++++++++++++++++++++++++
 1 file changed, 52 insertions(+)

diff --git a/nm-connection-editor.appdata.xml.in b/nm-connection-editor.appdata.xml.in
index 5f06de01..4f49ee24 100644
--- a/nm-connection-editor.appdata.xml.in
+++ b/nm-connection-editor.appdata.xml.in
@@ -27,4 +27,56 @@
   <developer_name>The NetworkManager Developers</developer_name>
 
   <content_rating type="oars-1.0" />
+
+  <releases>
+    <release version="1.8.24" date="2019-10-09" />
+    <release version="1.8.22" date="2019-05-29" />
+    <release version="1.8.20" date="2019-02-07" />
+    <release version="1.8.18" date="2018-09-07" />
+    <release version="1.8.16" date="2018-08-07" />
+    <release version="1.8.14" date="2018-06-25" />
+    <release version="1.8.12" date="2018-05-28" />
+    <release version="1.8.10" date="2017-12-19" />
+    <release version="1.8.8" date="2017-12-18" />
+    <release version="1.8.6" date="2017-11-07" />
+    <release version="1.8.4" date="2017-09-20" />
+    <release version="1.8.2" date="2017-06-13" />
+    <release version="1.8.0" date="2017-05-10" />
+    <release version="1.4.6" date="2017-03-06" />
+    <release version="1.4.4" date="2017-01-25" />
+    <release version="1.4.2" date="2016-09-30" />
+    <release version="1.4.0" date="2016-08-24" />
+    <release version="1.2.4" date="2016-08-03" />
+    <release version="1.2.2" date="2016-05-11" />
+    <release version="1.2.0" date="2016-04-20" />
+    <release version="1.0.10" date="2015-12-23" />
+    <release version="1.0.8" date="2015-11-20" />
+    <release version="1.0.6" date="2015-08-27" />
+    <release version="1.0.4" date="2015-07-14" />
+    <release version="1.0.2" date="2015-05-05" />
+    <release version="1.0.0" date="2014-12-18" />
+    <release version="0.9.10" date="2014-07-03" />
+    <release version="0.9.8.10" date="2014-04-24" />
+    <release version="0.9.8.9" date="2014-04-02" />
+    <release version="0.9.8.8" date="2013-11-11" />
+    <release version="0.9.8.4" date="2013-09-13" />
+    <release version="0.9.8.2" date="2013-06-07" />
+    <release version="0.9.8.0" date="2013-02-20" />
+    <release version="0.9.6.4" date="2012-10-26" />
+    <release version="0.9.6.2" date="2012-08-09" />
+    <release version="0.9.6.0" date="2012-08-07" />
+    <release version="0.9.4.1" date="2012-03-23" />
+    <release version="0.9.4.0" date="2012-03-23" />
+    <release version="0.9.2" date="2011-11-09" />
+    <release version="0.9.0" date="2011-08-22" />
+    <release version="0.8.4" date="2011-04-20" />
+    <release version="0.8.2" date="2010-11-03" />
+    <release version="0.8.1" date="2010-07-21" />
+    <release version="0.8" date="2010-02-18" />
+    <release version="0.7.2" date="2009-11-23" />
+    <release version="0.7.1" date="2009-04-13" />
+    <release version="0.7.0" date="2008-11-27" />
+    <release version="0.6.6" date="2008-03-06" />
+    <release version="0.6.5" date="2007-04-11" />
+  </releases>
 </component>
-- 
2.23.0

